grid.cpp

```diff
@@ -224,7 +232,7 @@ void initializeGrids(
       phiprof::start("Read restart");
       if (readGrid(mpiGrid,perBGrid,EGrid,technicalGrid,P::restartFileName) == false) {
          logFile << "(MAIN) ERROR: restarting failed" << endl;
-         exit(1);
+         exit(ExitCodes::RESTART_READ_FAILURE); // Do these have automatically recoverable failure modes?
```

Comment:
Uhm, like what?

Reply:
... like picking the previous restart file. If the restart file is bust, that would be the recovery, but do we even want to do that automatically?

Reply:
I think that's the kind of think better handled in the job script upon parsing the return code.

Reply:
Yup, well, that's why it is here anyway.